### PR TITLE
add insert ui

### PR DESCRIPTION
**Pull request by Tumao727**
### import part:
import

### preview part:
**1. no  field names**
preview-no

**2. with  field names**
preview-yes

### status part:
**1. loading**
status-loading

**2. success**
status-success

**3. fail**
status-error

### CODE REVIEW

File: client/src/components/grid/Table.tsx [hunk 1]
```diff
@@ -111,9 +112,11 @@ const EnhancedTable: FC<TableType> = props => {
     openCheckBox = true,
     disableSelect,
     noData,
-    showHoverStyle,
+    showHoverStyle = true,
```
Comment (shanghaikid) [inline client/src/components/grid/Table.tsx:115]: please write more comments about the properties.

File: client/src/components/insert/Container.tsx [hunk 2]
```diff
@@ -51,30 +52,37 @@ const InsertContainer: FC<InsertContentProps> = ({
...
-  const { handleCloseDialog } = useContext(rootContext);
+  const { handleCloseDialog, openSnackBar } = useContext(rootContext);
   const [activeStep, setActiveStep] = useState<InsertStepperEnum>(
     InsertStepperEnum.import
   );
   const [insertStatus, setInsertStauts] = useState<InsertStatusEnum>(
     InsertStatusEnum.init
   );
   // const [nextDisabled, setNextDisabled] = useState<boolean>(false);
+  const [collectionValue, setCollectionValue] =
```
Comment (shanghaikid) [inline client/src/components/insert/Container.tsx:63]: add comments about those status.

File: client/src/components/insert/Container.tsx [hunk 3]
```diff
@@ -98,13 +106,36 @@ const InsertContainer: FC<InsertContentProps> = ({
...
+  };
+
   const handleInsertData = () => {
     // mock status change
     setInsertStauts(InsertStatusEnum.loading);
     handleInsert();
     setTimeout(() => {
       setInsertStauts(InsertStatusEnum.success);
-    }, 500);
+    }, 1000);
```
Comment (shanghaikid) [inline client/src/components/insert/Container.tsx:138]: why 1000?

**Review by shanghaikid — CHANGES_REQUESTED** [review]

File: client/src/components/insert/Container.tsx (hunk 3 above)
Comment (Tumao727) [reply client/src/components/insert/Container.tsx]: There use `setTimout` just to mock async event, will be deleted later.

Comment (shanghaikid) [reply client/src/components/insert/Container.tsx]: ok. cool

**Review by nameczz — APPROVED** [review]

**Review by congtan0224 — APPROVED** [review]

**Review by shanghaikid — APPROVED** [review]

**Pull request merged**